### ISSUE TITLE
Windows client does not show up in the tray after system startup

### ISSUE THREAD

**Issue by fmpedro** [netbirdio/netbird#2692]
Even though I have the "Connect on Startup" option selected, the NetBird windows client will not show up in the system tray, as if it is not running. However, attempting to connect to systems that require the VPN connection to be active work, so that seems to indicate that the client is running in the background.
Attempting to open the app from the Start menu does result in the client showing up in the system tray.

I'm **not** using NetBird Cloud.

**NetBird version**
`v0.29.4`

**Comment by mgarces** [comment]
@fmpedro what is the version of Windows you are experiencing this issue?

**Comment by fmpedro** [comment]
Sorry, forgot to mention that. I'm running Windows 11 Pro, Version 10.0.22631 Build 22631

**Comment by fmpedro** [comment]
Yes, I have the same issue in ll the versions I've tested so far

**Comment by flotpg** [comment]
Unfortunately the feature is only available in the setup.**exe** but not the MSI.
Would it be possible to implement it for the MSI version as well?

**Comment by pnstafe** [comment]
Any update on this?

**Comment by fmpedro** [comment]
I'm currently not experiencing this problem anymore. FYI I'm currently on version 0.41 but the behaviour already dissapeared some versions ago
